src/app/spec/component-fixture-spec.helper.ts

```diff
@@ -0,0 +1,119 @@
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {
+    ChangeDetectorRef, DebugElement, SchemaMetadata, Type, Component, ViewChild,
+    ViewContainerRef, ComponentFactoryResolver, Optional, NgModule
+} from '@angular/core';
+import {SelectSpecHelper} from './select-spec.helper';
+import {Predicate} from '@angular/core/src/debug/debug_node';
+import {BrowserDynamicTestingModule} from '@angular/platform-browser-dynamic/testing';
+
+@Component({
+    selector: 'wave-test-host',
+    template: `<ng-template #component></ng-template>`
+})
+class TestHostComponent {
+    @ViewChild('component', {read: ViewContainerRef}) component: ViewContainerRef;
+
+    constructor(private componentFactoryResolver: ComponentFactoryResolver) {}
+
+    public load<T>(type: Type<T>, inputs: any): T {
```

Comment:
Maybe you can specify the type of `inputs` more precisely, since you iterate over it.